modules/glimpseBidAdapter.js

```diff
@@ -138,11 +138,16 @@ function getReferer(bidderRequest) {
 function getGdprConsentChoice(bidderRequest) {
   const hasGdprConsent =
     hasValue(bidderRequest) &&
-    hasValue(bidderRequest.gdprConsent) &&
-    hasStringValue(bidderRequest.gdprConsent.consentString)
+    hasValue(bidderRequest.gdprConsent)
 
   if (hasGdprConsent) {
-    return bidderRequest.gdprConsent
+    const gdprConsent = bidderRequest.gdprConsent
+
+    return {
+      consentString: gdprConsent.consentString || '',
+      vendorData: gdprConsent.vendorData || {},
+      gdprApplies: gdprConsent.gdprApplies || true,
```

Comment:
this is always true - might not be what you intended.

Reply:
It was added to capture the case where `gdprApplies` may be undefined.

Reply:
Ahh, thanks, it was just added to catch the case where `gdprApplies` is undefined. Opened a PR with the change (https://github.com/prebid/Prebid.js/pull/7783)